config_parser/config_reader.py

```diff
@@ -1,14 +1,22 @@
 """
-Module for getting value of certain parameter in certain section in cfg. file
+
+Module for getting value of certain parameter in certain section in cfg. file.
+Uses ConfigParser module for Python.
+
 """
 import ConfigParser
 
 
 def get_setting(cfg_file, section,  parameter):
     """
-    cfg_file - name or path and name of cfg file for parsing
-    section - name of section of parameter
-    parameter - name of parameter of value has to be received
+    Args:
+        cfg_file - name or path and name of cfg file for parsing
```

Comment:
As you know in cfg file value can be any type. So function can bring different types of data.